app/javascript/util/languageUtil.tsx

```diff
@@ -175,6 +176,16 @@ export function getReservedCommunityType(type: string | undefined): string {
   }
 }
 
+// Get the translated custom listing
+export function getCustomListingType(type: string | undefined): string {
+  switch (type) {
+    case CUSTOM_LISTING_TYPES.EDUCATOR_ONE:
```

Comment:
Are we only supporting Educator One Listing types currently? I see there are more options in Salesforce.